tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -563,8 +563,23 @@ public function checkOrderInvoicesCount(string $orderReference, int $expectedCou
     public function editProductsToOrderWithFollowingDetails(string $productName, string $orderReference, TableNode $table)
     {
         $orderId = SharedStorage::getStorage()->get($orderReference);
-        $productOrderDetail = $this->getOrderDetailFromOrder($productName, $orderReference);
         $data = $table->getRowsHash();
+        //if (!$data['price_tax_incl']) {
```

Comment:
Why the commented code? It should be removed?

Reply:
Why not using `getOrderDetailFromOrder` like before? Especially since you improved it?

Reply:
Yes, I should remove it.

Because `getOrderDetailFromOrder` use `getProductByName` which use `SearchProductsHandler` which change the `contextStateManager->setShop`. While `OrderController::updateProductAction` push directly the information.